CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in the last 3 major versions of the postg
 
 ## Unreleased
 
+## 8.1.1 - *2021-01-11*
+
+- Fix attribute updates for users with dashes
+
```

Comment:
Please don't set a version in here. Our auto release system will take care of this automatically. Please just put it below ``## Unreleased``.

Reply:
Fixed. This is different from my last contribution, https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD explicitly says I should not touch it. 😅

Anyway, let me know if there's more!